dom/src/main/java/org/apache/james/mime4j/field/DateTimeFieldLenientImpl.java

```diff
@@ -19,45 +19,41 @@
 
 package org.apache.james.mime4j.field;
 
-import org.apache.james.mime4j.codec.DecodeMonitor;
-import org.apache.james.mime4j.dom.FieldParser;
-import org.apache.james.mime4j.dom.field.DateTimeField;
-import org.apache.james.mime4j.stream.Field;
-
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Collection;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
 import java.util.Locale;
 import java.util.TimeZone;
 
+import org.apache.james.mime4j.codec.DecodeMonitor;
+import org.apache.james.mime4j.dom.FieldParser;
+import org.apache.james.mime4j.dom.field.DateTimeField;
+import org.apache.james.mime4j.stream.Field;
+
 /**
  * Date-time field such as <code>Date</code> or <code>Resent-Date</code>.
  */
 public class DateTimeFieldLenientImpl extends AbstractField implements DateTimeField {
 
     private static final String[] DEFAULT_DATE_FORMATS = {
-            "EEE, dd MMM yyyy HH:mm:ss ZZZZ",
-            "dd MMM yyyy HH:mm:ss ZZZZ",
-            "EEE, dd MMM yyyy HH:mm:ss.SSS 0000"};
+        "EEE, dd MMM yy HH:mm:ss ZZZZ",
+        "dd MMM yy HH:mm:ss ZZZZ",
+        "EEE, dd MMM yy HH:mm:ss.SSS 0000",
+        "EEE, dd MMM yyyy HH:mm:ss ZZZZ",
+        "dd MMM yyyy HH:mm:ss ZZZZ",
+        "EEE, dd MMM yyyy HH:mm:ss.SSS 0000"};
 
     private final List<String> datePatterns;
 
     private boolean parsed = false;
     private Date date;
 
-    DateTimeFieldLenientImpl(final Field rawField,
-                             final Collection<String> dateParsers, final DecodeMonitor monitor) {
+    private DateTimeFieldLenientImpl(Field rawField, DecodeMonitor monitor) {
```

Comment:
Do we remove the initial "dateParsers"? Is that Ok if we still would like to add more dateParsers? as I know the default date format in EN should be "MMM dd yyyy" instead of.